README.md

```diff
@@ -7,7 +7,24 @@ working with
 written in Go. Its primary use is to fetch and print OAuth 2.0 access
 tokens, which can be used with other command-line tools and shell scripts.
 
-## Previous Version
+## July 2019 Update
```

Comment:
Let's not add this here, we can create a new release after this one and add this to the release log.

Reply:
Ok. Let's create a proper tagged release after this change. Reverting this part of the read me for now.